core/util/extendConfig.js

```diff
@@ -59,17 +62,25 @@ function htmlReport(config, userConfig) {
   config.compareReportURL = path.join(config.html_report, 'index.html');
 }
 
-function comparePaths(config) {
+function comparePaths (config) {
   config.comparePath = path.join(config.backstop, 'compare');
   config.tempCompareConfigFileName = temp.path({suffix: '.json'});
 }
 
-function captureConfigPaths(config) {
-  config.captureConfigFileName = path.join(config.backstop, 'capture', 'config.json');
+function captureConfigPaths (config) {
+  var captureDir = path.join(tmpdir, 'capture');
+  if (!fs.existsSync(captureDir)) {
+    fs.mkdirSync(captureDir);
+  }
+  var tempCompareConfigFileName = config.tempCompareConfigFileName;
+  config.tempCompareConfigFileName = '';
+  var configHash = hash(config);
+  config.tempCompareConfigFileName = tempCompareConfigFileName;
```

Comment:
I am able to reproduce the error on my windows machine. I'll take a look tomorrow. Thank you for noticing this. I couldn't thought about a scenario for execution of same test at the same time. (still I can't :) )